package-lock.json

```diff
@@ -1961,9 +1953,9 @@
       "dev": true
     },
     "bn.js": {
-      "version": "5.1.2",
-      "resolved": "https://registry.npmjs.org/bn.js/-/bn.js-5.1.2.tgz",
-      "integrity": "sha512-40rZaf3bUNKTVYu9sIeeEGOg7g14Yvnj9kH7b50EiwX0Q7A6umbvfI5tvHaOERH0XigqKkfLkFQxzb4e6CIXnA==",
```

Comment:
Not sure if the package-lock.json should be completely merged since it seems to be reverting to older package versions

Reply:
Good catch, the package-lock.json file accidentally go overwritten by an older version. I will revert it.